test/QueryPlannerTest.cpp

```diff
@@ -2981,3 +2981,53 @@ TEST(QueryPlanner, Exists) {
       "GRAPH ?g { ?u ?v ?c}}}",
       filter);
 }
+
+// _____________________________________________________________________________
+TEST(QueryPlanner, PropertyPathWithGraph) {
```

Comment:
Do you have a test cases for a non-empty transitive path, e.g.
```
SELECT * FROM <g> { ?x <blubb>+ ?y }
```
This case should also correctly forward the graph constraint to the index scan for `<blubb>`.